src/hooks/data/useVaultApy.ts

```diff
@@ -76,9 +161,167 @@ export function useVaultTotalApr({
+    queries: [
+      {
+        queryKey: [_address, "auraFinance"],
+        queryFn: async () => await fetchApiAuraFinance(_address),
+        staleTime: Infinity,
+        enabled:
+          (apiQuery.isError || apiQuery.data === null) && !isToken && !isCurve,
+      },
+      {
+        queryKey: [_address, "auraMint"],
+        queryFn: async () => await getAuraMint(),
+        staleTime: Infinity,
+        enabled:
+          (apiQuery.isError || apiQuery.data === null) && !isToken && !isCurve,
+      },
+    ],
+  })
+
+  const extraTokenAwards = auraQuery?.[0].data?.extraTokenAwards
+  const swapFee = auraQuery?.[0].data?.swapFee
+  const auraMint = auraQuery?.[1].data
+
+  const balancerTotalAprFallbackQuery = useQuery(
+    [_address, "balancerTotalAprFallback", extraTokenAwards, swapFee],
+    {
+      queryFn: async () =>
+        await getBalancerTotalAprFallback(
+          _address,
+          extraTokenAwards,
+          swapFee,
+          auraMint
+        ),
+      retry: false,
+      enabled:
+        (apiQuery.isError || apiQuery.data === null) &&
+        !!swapFee &&
+        !!extraTokenAwards &&
+        !!auraMint &&
+        !isToken &&
+        !isCurve,
+    }
+  )
+  // END OF BALANCER
+
+  // TOKEN
+  const registryQuery = useContractRead({
+    ...useRegistryContract(),
+    functionName: "getTokenCompounderSymbol",
+    args: [_address ?? "0x"],
+    enabled: (apiQuery.isError || apiQuery.data === null) && isToken,
+  })
+
+  const ybTokenSymbol = registryQuery.data
+
+  const auraTokenQuery = useQuery([_address, "auraMint"], {
+    queryFn: async () => await getAuraMint(),
+    retry: false,
+    enabled:
+      (apiQuery.isError || apiQuery.data === null) &&
+      isToken &&
+      !!ybTokenSymbol &&
+      ybTokenSymbol === "fort-auraBAL",
+  })
+
+  const auraTokenMint = auraTokenQuery.data
+
+  const fortAuraBalAprFallback = useQuery(
+    [_address, "fortAuraBalAprFallback"],
+    {
+      queryFn: async () => await getFortAuraBalAprFallback(auraTokenMint),
+      retry: false,
+      enabled:
+        !!ybTokenSymbol && !!auraTokenMint && ybTokenSymbol === "fort-auraBAL",
+    }
+  )
+
+  const fortCvxCrvAprFallback = useQuery([_address, "fortCvxCrvAprFallback"], {
+    queryFn: async () => await getFortCvxCrvAprFallback(),
+    retry: false,
+    enabled: !!ybTokenSymbol && ybTokenSymbol === "fort-cvxCRV",
+  })
+
+  const glpQuery = useContractRead({
+    abi: glpRewardsDistributorAbi,
+    address: GLP_REWARDS_DISTRIBUTOR_ADDRESS as `0x${string}`,
+    functionName: "tokensPerInterval",
+    enabled: !!ybTokenSymbol && ybTokenSymbol === "fortGLP",
+  })
+
+  const ethRewardsPerSecond = glpQuery.data
+
+  const fortGlpAprFallback = useQuery([_address, "fortGlpAprFallback"], {
+    queryFn: async () => await getFortGlpAprFallback(ethRewardsPerSecond),
+    retry: false,
+    enabled:
+      !!ybTokenSymbol && !!ethRewardsPerSecond && ybTokenSymbol === "fortGLP",
+  })
+  // END OF TOKEN
+
+  if (
```

Comment:
I think we probably want to check for the type of asset we're returning data for in these conditionals.